code/modules/shuttle/shuttle.dm

```diff
@@ -1161,6 +1168,19 @@
 /obj/docking_port/mobile/emergency/on_emergency_dock()
 	return
 
+///self destruct if this is returned in process
+#define SHUTTLE_EVENT_CLEAR 2
```

Comment:
Having this defined twice (once here, once in the next file) isn't nice, should go into a relevant defines file